.github/workflows/workflow.yml

```diff
@@ -1,29 +1,23 @@
 name: Test and Deploy
-on:
-  pull_request:
-    branches: 
-      - master
-  push:
-    branches:
-      - master
-  create:
```

Comment:
Now it won't run on tagging, breaks logic.

Reply:
A tag counts as a push iirc, no?

Reply:
I'll add tags back in tho